@AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/mpd/fn_mpdLWPNDraw.sqf

```diff
@@ -188,10 +188,17 @@ if (_curWpn isKindOf ["fza_hellfire", configFile >> "CfgWeapons"]) then {
 		_tex = switch (_salType) do {
 			case "sal1": {"\fza_ah64_us\tex\icons\sal1.paa"};
 			case "sal2": {"\fza_ah64_us\tex\icons\sal2.paa"};
-			case "rf": {"\fza_ah64_us\tex\icons\sal2.paa"};
+			case "rf": {""};
 			default {""};
 		};
 		_heli setObjectTexture [SEL_MPD_PL_HF_SALT, _tex];
+		_tex = switch (_salType) do {
+			case "sal1": {"\fza_ah64_us\tex\icons\sal.paa"};
+			case "sal2": {"\fza_ah64_us\tex\icons\sal.paa"};
+			case "rf": {"\fza_ah64_us\tex\icons\salRF.paa"};
```

Comment:
Just a real nitpick, this texture shouldn't be called `salRF.paa`, as SAL means "Semi Active Laser" so what that means is "Semi Active Laser Radio Frequency" when in reality it should probably just be `RF`